src/routes/Landing.tsx

```diff
@@ -1,18 +1,60 @@
 import React, { Fragment } from 'react';
+import { useFlag } from '@unleash/proxy-client-react';
 
 import '../components/app-content-renderer/styles/panels.scss';
 
 import FirstPanel from '../components/app-content-renderer/first-panel';
 import SecondPanel from '../components/app-content-renderer/second-panel';
 import VirtualAssistant from '../components/app-content-renderer/virtual-assistant';
+import { useLoadModule } from '@scalprum/react-core';
+import { PageSection } from '@patternfly/react-core/dist/dynamic/components/Page';
+
+const getWidgetLayoutLandingPage = () => {
+  const scope = 'frontendStarterApp';
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  const [{ default: GridLayout }] = useLoadModule(
```

Comment:
We don't need to use `useLoadModule` instead we could use `ScalprumComponent` and pass `scope` and `module` to it.